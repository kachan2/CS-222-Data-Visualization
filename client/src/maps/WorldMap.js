import React from "react";
// import ReactDOM from "react-dom";
import { ComposableMap, Geographies, Geography } from "react-simple-maps";

import "./styles.css";

const WorldMap = () => (
  <div>
    <ComposableMap>
      <Geographies geography="/features.json">
        {({ geographies }) =>
          geographies.map((geo) => (
            <Geography key={geo.rsmKey} geography={geo} />
          ))
        }
      </Geographies>
    </ComposableMap>
  </div>
);

export default WorldMap;