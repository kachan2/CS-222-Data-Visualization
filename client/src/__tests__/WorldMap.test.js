import React from "react";
import ShallowRenderer from 'react-test-renderer/shallow';

// Components
import WorldMap from "../maps/WorldMap"

it('renders correctly react-test-renderer', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<WorldMap />);
  const result = renderer.getRenderOutput();

  expect(result).toMatchSnapshot();
});