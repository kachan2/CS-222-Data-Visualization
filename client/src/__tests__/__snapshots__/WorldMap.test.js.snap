// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly react-test-renderer 1`] = `
<div>
  <ComposableMap>
    <Geographies
      geography="/features.json"
    >
      [Function]
    </Geographies>
  </ComposableMap>
</div>
`;
